### DISCUSSION TITLE
Trix field with files does not work in Action fields

### DISCUSSION THREAD

**Discussion by webard** [laravel/nova-issues#6393] [Ideas & Feature Requests] [answered]
- Laravel Version: 11.0.7
- Nova Version: 4.33.3
- PHP Version: 8.2.19
- Database Driver & Version: MySQL 8.3.0
- Operating System and Version: MacOS 14.5
- Browser type and version: Chrome 124.0.6367.208 

### Description:

Action with `Trix` field which enabled file upload does not upload files correctly.

`Action` is appended to `User` resource:
```php
<?php

namespace App\Nova\Actions;

use Laravel\Nova\Actions\Action;
use Laravel\Nova\Fields\Trix;
use Laravel\Nova\Http\Requests\NovaRequest;

class TestAction extends Action
{
    public function fields(NovaRequest $request)
    {
        return [
            Trix::make('Reason')->withFiles('public')
        ];
    }
}
```

After pasting image to Trix field in Nova, there is 404 error:
```
Failed to load resource: the server responded with a status of 404 (Not Found): http://127.0.0.1:8000/nova-api/users/field-attachment/reason
```

Uploading when Trix field is added to Resource works correctly.

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Create Action
2. Append it to any Resource
3. Add `Trix` field to Action and enable upload by `withFiles()` method
4. Try to paste image to Trix field on Nova UI

**Comment by webard** [comment]
@crynobone I don't understand why bug report was moved to discussion. Trix field is not beta feature, documentation says nothing about incompatibility with Actions, so what is the reason?

**Reply by crynobone** [reply, marked as answer]
Attachment fields explicitly limited to Resources, so this goes to Ideas and Feature Requests.
